app/models/work.rb

```diff
@@ -360,11 +360,6 @@ def pre_curation_uploads_fast
     s3_query_service.client_s3_files
   end
 
-  # Fetches the data from S3 directly bypassing ActiveStorage
-  def pre_curation_uploads_fast
-    s3_query_service.client_s3_files
-  end
-
```

Comment:
@carolyncole This method was duplicated. I removed the duplicate **there** is still one `pre_curation_uploads_fast` in the code.